lib/dlss/capistrano.rb

```diff
@@ -3,3 +3,4 @@
 require 'capistrano/shared_configs'
 
 Dir.glob("#{__dir__}/capistrano/tasks/*.rake").each { |r| import r }
+Dir.glob("#{__dir__}/capistrano/tasks/setup/*.rake").each { |r| import r }
```

Comment:
Make sure the setup task is loaded *after* all of the other tasks so that they may be enhanced with a new prerequisite.